Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Bijectors"
 uuid = "76274a88-744f-5084-9051-94815aaf08c4"
-version = "0.12.3"
+version = "0.12.4"
```

Comment:
I think we need to update the Distributions compat. AFAIK I (?) added ReshapedDistribution relatively recently.

Reply:
Ah yes, good catch!